torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -142,11 +143,7 @@ struct ConstantPropagator {
       bool aliasing_types,
       bool ignore_custom_classes)
       : graph_(std::move(graph)) {
-    if (aliasing_types) {
-      aliasDb_ = torch::make_unique<AliasDb>(graph_);
-    } else {
-      aliasDb_ = nullptr;
-    }
+    aliasing_types_ = aliasing_types;
```

Comment:
what's the rationale behind removing `torch::make_unique`?

Reply:
it's to lazy initialize alias db, since constructing it can be slow. if you're optimizing a graph with all scalars you dont need to access an aliasdb to constant propagate